chapters/en/chapter11/5.mdx

```diff
@@ -0,0 +1,243 @@
+# Evaluation
+
+With a finetuned model through either SFT or LoRA SFT, we should evaluate it on standard benchmarks.
+
+## Automatic Benchmarks
+
+Automatic benchmarks serve as standardized tools for evaluating language models across different tasks and capabilities. While they provide a useful starting point for understanding model performance, it's important to recognize that they represent only one piece of a comprehensive evaluation strategy.
+
+## Understanding Automatic Benchmarks
```

Comment:
Maybe good to mention popular benchmarks for some domains like:

* Code = HumanEval
* MATH
* AlpacaEval2 for chat